pom.xml

```diff
@@ -2899,7 +2899,6 @@
               <arg>-deprecation</arg>
               <arg>-feature</arg>
               <arg>-explaintypes</arg>
-              <arg>-target:jvm-1.8</arg>
```

Comment:
How about change to `<arg>-target:jvm-${java.version}</arg>` ?

Reply:
Yeah, that should work as well.